tests/automation_framework/data/worker/workerregister_workOrderPayloadFormats_JSON_RPC_JWT.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Rename the file to workerregister_workOrderPayloadFormats_JSONRPCJWT.json as per the naming conventions.